docs/_edot-sdks/php/setup/index.md

```diff
@@ -5,9 +5,103 @@ nav_order: 1
 parent: EDOT PHP
 ---
 
-# Setting up EDOT PHP
-
-TODO:
+<!-- TODO:
 - where to download
 - explicit description of basic setup here (even if it overlaps with upstream docs)
-- link to upstream docs for more advanced setup use cases
\ No newline at end of file
+- link to upstream docs for more advanced setup use cases -->
+
+# Setting up EDOT PHP
+
+This guide shows you how to use the Elastic Distribution of OpenTelemetry PHP (EDOT PHP) to instrument your PHP application and send OpenTelemetry data to an Elastic Observability deployment.
+
+**Already familiar with OpenTelemetry?** It's an explicit goal of this distribution to introduce _no new concepts_ outside those defined by the wider OpenTelemetry community.
+
+**New to OpenTelemetry?** This section will guide you through the _minimal_ configuration options to get EDOT PHP set up in your application. You do _not_ need any existing experience with OpenTelemetry to set up EDOT PHP initially. If you need more control over your configuration after getting set up, you can learn more in the [OpenTelemetry documentation](https://opentelemetry.io/docs/languages/php/).
+
+## Prerequisites
+
+Before you begin, make sure you have a destination for the telemetry data collected by EDOT PHP.
+While EDOT PHP can export data to any OpenTelemetry Protocol (OTLP)–compatible endpoint, this guide focuses on using [Elastic Observability](https://www.elastic.co/observability) as the backend.
+You can either use an existing Elastic Cloud deployment or create a new one.
+
+To quickly get up and running, follow the [Elastic OpenTelemetry Quickstart guide](https://elastic.github.io/opentelemetry/quickstart/), which walks you through:
+
+- Creating a free Elastic Cloud deployment
+- Configuring your OpenTelemetry agent
+- Exploring traces and metrics in Kibana
+
+## Install
+
+### Prerequisites
```

Comment:
we have a prerequisite section above, and one here, it's a bit confusing. Can we consolidate those?